src/main/webapp/app/exercises/programming/assess/programming-assessment-dashboard/programming-assessment-dashboard.component.html

```diff
@@ -0,0 +1,181 @@
+                    </th>
+                    <th jhiSortBy="result.assessmentType">
+                        <a class="th-link" jhiTranslate="artemisApp.assessment.dashboard.columns.assessmentType">Assessment Type</a>
+                        <fa-icon [icon]="'sort'"></fa-icon>
+                    </th>
+                    <th jhiSortBy="result.assessor.name">
+                        <a class="th-link" jhiTranslate="artemisApp.assessment.dashboard.columns.assessor">Reviewer</a>
+                        <fa-icon [icon]="'sort'"></fa-icon>
+                    </th>
+                    <th>
+                        <a class="th-link" jhiTranslate="artemisApp.assessment.dashboard.columns.assessmentFirst">Assessment</a>
+                    </th>
+                    <th *ngIf="numberOfCorrectionrounds == 2">
+                        <a class="th-link" jhiTranslate="artemisApp.assessment.dashboard.columns.assessmentSecond">Assessment</a>
+                    </th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr *ngFor="let submission of filteredSubmissions; let i = index">
+                    <td>{{ i + 1 }}</td>
+                    <td>{{ submission.submissionDate | artemisDate }}</td>
+                    <td>
+                        <jhi-result [participation]="submission.participation"></jhi-result>
+                    </td>
+                    <td>
+                        <ng-container *ngIf="submission.participation.submissions">
+                            {{ submission.participation.submissions.length }}
+                        </ng-container>
+                    </td>
+                    <td>
+                        <span *ngIf="submission.latestResult && submission.latestResult!.score != null"> {{ submission.latestResult!.score }} % </span>
+                    </td>
+                    <td>
+                        {{ submission.durationInMinutes }}
+                        {{ 'artemisApp.assessment.dashboard.minutes' | translate }}
+                    </td>
+                    <td>{{ assessmentTypeTranslationKey(submission.latestResult) | translate }}</td>
+                    <td>
+                        <span *ngIf="submission.latestResult && submission.latestResult!.assessor">{{ submission.latestResult!.assessor!.name }}</span>
+                    </td>
+                    <td *ngFor="let item of [].constructor(numberOfCorrectionrounds); let correctionRound = index">
+                        <span *jhiHasAnyAuthority="['ROLE_ADMIN', 'ROLE_INSTRUCTOR', 'ROLE_TA']">
+                            <!-- exercise without exam-->
+                            <span *ngIf="exercise.type === ExerciseType.PROGRAMMING && exercise.course && submission">
+                                <button *ngIf="submission.results.length == 0" [disabled]="busy" [routerLink]="getAssessmentLink(submission)" class="btn btn-success btn-sm mb-1">
```

Comment:
There was a pull request that changes buttons onto links, so wouldn't it be nice to follow that approach in this pull request as well.

Reply:
I changed all the buttons to links :)